module/Core/src/Core/Entity/AttachableEntityInterface.php

```diff
@@ -10,29 +10,27 @@
  */
 namespace Core\Entity;
 
-use Core\Entity\Collection\AttachedEntitiesCollection;
-
 interface AttachableEntityInterface
```

Comment:
I'm not so sure about this name anymore. it's a bit misleading since entities implementing this interface are NOT attachable itself, but ALLOW others entities to be attached to.... But I cannot think of a better suited name right now :(